app/views/dashboard.html

```diff
@@ -1 +1,17 @@
-<h1>Dashboard</h1>
\ No newline at end of file
+<h1>Dashboard</h1>
+
+<md-list>
+	<md-list-item class="md-2-line" ng-repeat="rule in vm.rules track by rule.id" ng-click="vm.editRule(rule)">
```

Comment:
Does the click do anything at the moment? `vm.editRule()` returns the rule id but can't quite see how it's used, or is it just in prep for for further changes?